README.md

```diff
@@ -76,7 +76,7 @@ The [`pulse`](./pulse) folder contains examples of projects using [Prisma Pulse]
 | [`sveltekit`](https://github.com/prisma/prisma-examples/tree/latest/typescript/sveltekit)                                     | [SvelteKit](https://kit.svelte.dev/) app using SvelteKit's [actions](https://kit.svelte.dev/docs/form-actions) and [load](https://kit.svelte.dev/docs/form-actions#loading-data) functions           |     | [SvelteKit](https://kit.svelte.dev/) app using SvelteKit's [actions](https://kit.svelte.dev/docs/form-actions) and [load](https://kit.svelte.dev/docs/form-actions#loading-data) functions |
 | [`trpc-nextjs`](https://github.com/prisma/prisma-examples/tree/latest/typescript/trpc-nextjs)                                 | [Next.js](https://nextjs.org/) app with [tRPC ](https://trpc.io/)                                                                                                                                    |
 | [`remix`](https://github.com/prisma/prisma-examples/tree/latest/typescript/remix)                                             | [Remix](https://remix.run/) app                                                                                                                                                                      |
-
+| [`nuxtjs-prisma-nuxt-module`](https://github.com/prisma/prisma-examples/tree/latest/typescript/nuxtjs-prisma-nuxt-module)                                             | A nuxt example app using the [Prisma Nuxt module](https://github.com/prisma/nuxt-prisma)                                                                                                                                                                     |
```

Comment:
`Nuxt` is standardizing on `Nuxt` rather than `Nuxt.js` etc
```suggestion
| [`nuxt-prisma-nuxt-module`](https://github.com/prisma/prisma-examples/tree/latest/typescript/nuxtjs-prisma-nuxt-module)                                             | A nuxt example app using the [Prisma Nuxt module](https://github.com/prisma/nuxt-prisma)                                                                                                                                                                     |
```